G1ANT.Addon.PDF/PdfOpenCommand.cs

```diff
@@ -30,11 +30,12 @@ public class Arguments : CommandArguments
 
         public void Execute(Arguments arguments)
         {
-            FileStream fs = File.Open(arguments.Path.Value, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+            MemoryStream memoryStream = new MemoryStream();
```

Comment:
using (FileStream fs = ....)
{
  var memoryStream = ...
  fs.CopyTo...
  ... rest of the implementation
}